driver-mct/cime_config/config_component_e3sm.xml

```diff
@@ -270,6 +270,7 @@
       <value compset="_MPASSI%BGC.*_MPASO%OIECO.*_BGC%BCRD" >CO2C_OI</value>
       <value compset="_MPASSI%BGC.*_MPASO%OIECO.*_BGC%BDRC" >CO2C_OI</value>
       <value compset="_MPASSI%BGC.*_MPASO%OIECO.*_BGC%BDRD" >CO2C_OI</value>
+      <value compset="_BGC%LNDATM">CO2B</value>
```

Comment:
Does this change belong in this PR?

Reply:
Oops.  No, I added these to do more testing with this sort of configuration, but they aren't necessary.